src/main/resources/camunda/applicant_trial_ready_notify_others.bpmn

```diff
@@ -17,24 +17,23 @@
       <bpmn:incoming>Flow_1ui9efm</bpmn:incoming>
       <bpmn:outgoing>Flow_1hce35l</bpmn:outgoing>
     </bpmn:callActivity>
-    <bpmn:serviceTask id="OtherTrialReadyNotifyRespondentSolicitor1" name="Notify Respondent solicitor 1" camunda:type="external" camunda:topic="processCaseEvent">
+    <bpmn:serviceTask id="ApplicantTrialReadyNotifierOthers" name="Notify Respondent solicitors" camunda:type="external" camunda:topic="processCaseEvent">
```

Comment:
since name and action do not match - i think its notify applicant solicitors not respondent. please check